docs/operations/auth-ldap.md

```diff
@@ -187,6 +187,8 @@ curl -i -v  -H "Content-Type: application/json" -u internal -X POST http://loca
 
 ### Step 5. Assign the role to the user 
 
+Note: This step is only needed if either you don't want to rely on the groups as configured in your ldap server, or if your ldap server does not support the `memberOf` attribute, or if for example the ldap server does not support recursive group membership. Druid will use any roles explicitly assigned to the user, which can be setup in this step.
```

Comment:
There are three conditions, is that right? Or the third is an example?  If three, it might be good to break out into bullets.. I also would suggest putting the last sentence first, since it explains the goal.... something like: 

```suggestion
In this step, you can configure Druid to apply specific roles to users. This step is only needed in the following cases: 

- You don't want to rely on group associations configured in your LDAP server,
- Your LDAP server does not support the `memberOf` attribute, or 
- The LDAP server does not support recursive group membership. 

If this is not the case for your scenario, you can skip this step. 
```